finalPrep/README.md

```diff
@@ -18,37 +18,37 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [*] I have refactored my `section1` exercises to the best of my ability
+- [*] I have refactored my `section2` exercises to the best of my ability
+- [*] I have refactored my `section3` exercises to the best of my ability
+- [*] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [*] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [*] Study/Project work time
+- [*] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![alt text](https://user-images.githubusercontent.com/21073095/114811918-3a97fc80-9d6c-11eb-85f5-f001e9781c28.png)
```

Comment:
It looks like you have an appropriate respect for the amount of time Turing will take. It looks like you want to prioritize family time on weekends, which is great and needed, but will also throw it out there that many project teams may want to work/meet on weekends, so I just encourage over communication, and you may need to play some weeks by ear, or allow for a bit more work some weekends than others.